package.json

```diff
@@ -14,38 +14,49 @@
     "react-native",
     "driver"
   ],
+  "files": [
+    "lib"
+  ],
   "main": "lib/cjs/index.js",
   "typings": "lib/cjs/index.d.ts",
-  "types": "lib/cjs/index.d.ts",
   "dependencies": {
-    "@cycle/react": "1.4.x",
-    "react": "16.4.x",
-    "react-native": ">=0.55.0",
-    "xstream": "11.x.x"
+    "@cycle/react": "2.x",
+    "@cycle/run": "5.x",
```

Comment:
`cycle/run` is used in tests, not in `src`, so I would not put it in `dependencies`, it should be only in `devDependencies`.